spec/controllers/chess_pieces_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe ChessPiecesController, type: :controller do
+
+
+    describe "chess_pieces#update" do
+        it "should successfully update chess piece location" do
```

Comment:
Need a test for what happens if they somehow send a bad move, how does that response look like?